### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by emagdalenaC2i**
@mbartlett21 Add your contact information.

I do not know if you used the nDraft FS code as source but If you want you can add nDraft to the copyright.

I guess you do not have problems that I also figure in the copyright

### CODE REVIEW

**Comment by mbartlett21** [comment]
I created my own FS, not using code from nDraft.

**Comment by emagdalenaC2i** [comment]
So, if you agree that I also figure in the copyright you can merge and copy & paste to Onshape

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2018 Morgan Bartlett, Contract-Creativity (https://contract-creativity.com/)
```
Comment (emagdalenaC2i) [inline LICENSE]: Copyright (c) 2018 Contract-Creativity (https://contract-creativity.com/): Morgan Bartlett

Comment (emagdalenaC2i) [reply LICENSE]: We use to add the company first in other projects I use to collaborate

**Comment by mbartlett21** [comment]
I agree 😄

**Pull request merged**